osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/valueobjects/smartmetering/EventLogCategory.java

```diff
@@ -15,8 +15,7 @@
     FRAUD_DETECTION_LOG,
     COMMUNICATION_SESSION_LOG,
     M_BUS_EVENT_LOG,
-    POWER_QUALITY_EVENT_LOG,
-    POWER_FAILURE_EVENT_LOG;
+    POWER_QUALITY_EVENT_LOG;

```

Comment:
Besides this change, that is tied to the decision between outages and power failure event logs, I noticed someone messed up the line endings of this file a year and four months ago.
Line endings should only be newlines, not Windows style carriage return plus newline.